pkg/webhook/v1beta1/experiment/validation_webhook.go

```diff
@@ -79,25 +78,6 @@ func (v *ExperimentValidator) Handle(ctx context.Context, req admission.Request)
 		}
 	}
 
-	// After metrics collector sidecar injection in Job level done, delete validation for namespace labels
```

Comment:
I'm not sure what `sidecar injection in Job level` means. This comment seemed to add in #828 3 years ago.
After that, @andreyvelich refactored the `getKatibJob` function (maybe, for job level injection?) in #1303.

@andreyvelich @johnugeorge Do you remember what this comment means?

Reply:
I don't think, this comment is still valid.

 In Job level, we look at type of Job(TFJob, PytorchJob etc) and decide the job pod which needs metrics collector injection. In Pod level, we check pod level labels and decide if we need to inject metrics collector or not.  Right now, we are doing pod level where inject metric collector in the pods that  contain PrimaryPodLabels or not.

Reply:
I see. Thanks for clarifying! @johnugeorge

Reply:
@tenzen-y I guess the idea behind Job level mutation was described here: https://github.com/kubeflow/katib/blob/master/docs/proposals/metrics-collector.md#mutating-webhook.
But since we dynamically support any kind of Kubernetes resources, we mutate only in the Pod level.

Reply:
Yes, I agree. I thought we cannot inject sidecars in Job level since we support any K8s resources, as you say.
Maybe, it would be good to remove the comment to avoid confusion.

Reply:
@johnugeorge @andreyvelich @tenzen-y Thanks for the explanation! I will close this pr